Add optional setting of pubsub credentials (#1985)
TheThingsNetwork/lorawan-stack: CHANGELOG.md

- Handling multiple consequent updates of MQTT pubsub/webhook integrations in the Console.
- Displaying total device count in application overview section when using device search in the Console
- FQDN used for Backend Interfaces interoperability requests.

> - Username and password not required in pubsub integration if tls is enabled

Reviewer: ```suggestion
- Changing username and password to be not required in pubsub integration
```
